modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_role/details.jsp

```diff
@@ -103,16 +103,16 @@ if (accountRole != null) {
 </liferay-frontend:edit-form>
 
 <c:if test="<%= role == null %>">
-	<aui:script require="frontend-js-web/liferay/debounce/debounce.es as debounceModule">
+	<aui:script require="frontend-js-web/index as frontendJsWeb">
+		var {debounce} = frontendJsWeb;
```

Comment:
TBH, I have no idea of what our JSP conventions are because I never touch code of that type. I simply adapted the code, but if there's a better way to do it, we can change it :+1: .

AFAIK we use ESM syntax everywhere since we stopped supporting IE, so I supposed we can use destructuring...

Reply:
@izaera thats fine by me. I did a quick search and saw some other cases where we use destructuring in jsp, so you should be fine